css-view-transitions-1/Overview.bs

```diff
@@ -1262,43 +1262,10 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		1. [=Queue a global task=] on the [=DOM manipulation task source=],
 			given |transition|'s [=relevant global object=],
-			to execute the following steps:
+			to [=call the update callback=].
 
 				Note: A task is queued here because the texture read back in [=capturing the image=] may be async,
 					although the render steps in the HTML spec act as if it's synchronous.
-
-			1. If |transition|'s [=ViewTransition/phase=] is "`done`", then abort these steps.
-
-				Note: This happens if |transition| was [=skip the view transition|skipped=] before this point.
```

Comment:
We still need to handle this case IIUC. With this PR, we queue a task to call the update callback. Before that task could run, author can call skipTransition() via script. skipTransition() will see that the phase is before "update-callback-called" and also post a task to call the update callback. And call the update callback always invokes the update callback, even if the phase is done because it assumes that we're in the skip case.

Reply:
Fair enough, reinstated this one.